src/main/groovy/life/qbic/portal/sampletracking/ui/SampleImport.groovy

```diff
@@ -0,0 +1,40 @@
+package life.qbic.portal.sampletracking.ui
+
+import com.vaadin.ui.Button
+import com.vaadin.ui.FormLayout
+import com.vaadin.ui.TextField
+import com.vaadin.ui.VerticalLayout
+import life.qbic.portal.sampletracking.app.PortletController
+
+class SampleImport extends VerticalLayout {
+    private PortletController controller
+    private SampleImportModel sampleImportModel
+
+    private Button singleSampleAddButton
+    private TextField additionalSampleId
+
+    SampleImport(PortletController controller, SampleImportModel sampleImportModel) {
+        super()
+        this.controller = controller
+        this.sampleImportModel = sampleImportModel
+        initLayout()
+    }
+
+    private def initLayout() {
+
+        FormLayout formLayout = new FormLayout()
```

Comment:
```suggestion
        final FormLayout formLayout = new FormLayout()
```

Reply:
No I would not do this. We still need to add the vaadin Components to this layout. This is not one of our classes and was meant to be a wrapper to the other vaadin Components (which was not implemented yet)